src/main/java/org/reaktivity/specification/nukleus/CoreFunctions.java

```diff
@@ -81,6 +84,35 @@ public static Random random()
         return array;
     }
 
+    @Function
+    public static byte[] capabilities(String capability, String... optionalCapabilities)
+    {
+        int capabilities = 0x00;
+        capabilities = maskBits(capability, capabilities);
+
+        for (int i = 0; i < optionalCapabilities.length; i++)
+        {
+            capabilities = maskBits(optionalCapabilities[i], capabilities);
+        }
+
+        ByteBuffer bb = ByteBuffer.allocate(1);
+        bb.putInt(capabilities);
+        return bb.array();
```

Comment:
This should overflow because an `int` is `4` bytes but we allocated `1`, and size `1` is actually correct.

```
bb.put((byte) capabilities);
```